packages/api/tests/lambdas/ems-distribution-report.js

```diff
@@ -216,5 +220,47 @@ test.serial('emsDistributionReport writes a correct report out to S3 when two re
   const logLines = getObjectResponse.Body.toString().split('\n');
 
   // Verify that the correct report was generated
-  t.deepEqual(logLines, expectedReportContent);
+  t.deepEqual(logLines, expectedReportContentByTime);
+});
+
+test.serial('emsDistributionReport writes multiple reports', async (t) => {
```

Comment:
Can we make this test name more descriptive? Something like… `emsDistributionReport writes multiple reports when report spans multiple days`?